teraserver/python/libtera/db/models/TeraDeviceType.py

```diff
@@ -36,15 +50,23 @@ def get_devices_types():
     def get_device_type(dev_type: int):
         return TeraDeviceType.query.filter_by(id_device_type=dev_type).first()
 
-    def get_name(self):
-        name = 'Inconnu'
-        if self.id_device_type == TeraDeviceType.DeviceTypeEnum.VIDEOCONFERENCE.value:
-            name = 'Vidéoconférence'
-        if self.id_device_type == TeraDeviceType.DeviceTypeEnum.SENSOR.value:
-            name = 'Capteur'
-        if self.id_device_type == TeraDeviceType.DeviceTypeEnum.ROBOT.value:
-            name = 'Robot'
-        if self.id_device_type == TeraDeviceType.DeviceTypeEnum.BUREAU_ACTIF.value:
-            name = 'Bureau Actif'
-        return name
+    @staticmethod
+    def get_device_type_by_id(dev_type: int):
+        return TeraDeviceType.query.filter_by(id_device_type=dev_type).first()
+
+    @staticmethod
+    def get_device_type_by_name(dev_type: str):
+        return TeraDeviceType.query.filter_by(device_type_name=dev_type).all()
+
+    @staticmethod
+    def get_device_type_by_key(dev_type: str):
+        return TeraDeviceType.query.filter_by(device_type_key=dev_type).first()
+
+    @staticmethod
+    def get_device_type_by_type(dev_type: str):
```

Comment:
Why, this method, it is exactly the same as get_device_type_by_key.

Reply:
Yes, forgot to take it out. To be deleted without issue.